staging/provisioning/windows/hostsconfigagent.ps1

```diff
@@ -0,0 +1,54 @@
+$Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
+$clusterFQDN = $Global:ClusterConfiguration.Kubernetes.ControlPlane.IpAddress
+$hostsFile="C:\Windows\System32\drivers\etc\hosts"
+$retryDelaySeconds = 15
+
+# Dot-source scripts with functions that are called in this script
+. c:\AzureData\k8s\kuberneteswindowsfunctions.ps1
```

Comment:
I am not sure if since these files are not signed we might run into issues.  @ksubrmnn do you know if we can use the non signed powershell here?

Reply:
if so, should I move it to `staging/provisioning/windows`?

Reply:
I think it depends on when the code is run. During provisioning vs after provisioning. We've only been signing code that runs after provisioning.

Reply:
removed this by copying the required functions.